doc/index.rst

```diff
@@ -245,6 +247,24 @@ On CI platform usually some variables are available for the convenience. Since a
 Still, you may want to pass some other vars. `DOCKER_RUN_OPTS='-e MY_VARIABLE_VALUE'` should do the trick.
 You can even set it to a specific value: `DOCKER_RUN_OPTS='-e MY_VARIABLE_VALUE=42'` (format varies per CI platform. These are Gitlab CI example).
 
+Re-use the container image
+--------------------------
+
+``industrial_ci`` builds a ``Docker`` image using the associated repository on the specified operating system per every job. While the built Docker image is thrown away once the job finishes by default, there's a way to access the built image post job so that you can re-use it.
+
+To do so, simply set ``DOCKER_COMMIT`` the name of the image of your choice. Then you'll be able to access that image. For example in your CI config (e.g. ``.travis.yml``), add something like ::
+
+  variables:
+      DOCKER_COMMIT=registry.gitlab.com/your-org/your-repo:your_img
+  :
+  script:
+      - docker push $DOCKER_COMMIT
+
+Note for the built Docker image:
```

Comment:
I would remove this note section. It is way more complicated and highl depends on the use case.